GVFS/GVFS.Hooks/GVFS.Hooks.Linux.csproj

```diff
@@ -0,0 +1,113 @@
+    -->
+    <Compile Include="..\GVFS.Common\ConsoleHelper.cs">
+      <Link>Common\ConsoleHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Git\GitConfigHelper.cs">
+      <Link>Common\Git\GitConfigHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Git\GitConfigSetting.cs">
+      <Link>Common\Git\GitConfigSetting.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Git\GitVersion.cs">
+      <Link>Common\Git\GitVersion.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\GVFSConstants.cs">
+      <Link>Common\GVFSConstants.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\GVFSEnlistment.Shared.cs">
+      <Link>Common\GVFSEnlistment.Shared.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\GVFSLock.Shared.cs">
+      <Link>Common\GVFSLock.Shared.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NamedPipes\BrokenPipeException.cs">
+      <Link>Common\NamedPipes\BrokenPipeException.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NamedPipes\LockNamedPipeMessages.cs">
+      <Link>Common\NamedPipes\LockNamedPipeMessages.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NamedPipes\NamedPipeClient.cs">
+      <Link>Common\NamedPipes\NamedPipeClient.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NamedPipes\NamedPipeStreamReader.cs">
+      <Link>Common\NamedPipes\NamedPipeStreamReader.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NamedPipes\NamedPipeStreamWriter.cs">
+      <Link>Common\NamedPipes\NamedPipeStreamWriter.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\NativeMethods.Shared.cs">
+      <Link>Common\NativeMethods.Shared.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Paths.Shared.cs">
+      <Link>Common\Paths.Shared.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\ProcessHelper.cs">
+      <Link>Common\ProcessHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\ProcessResult.cs">
+      <Link>Common\ProcessResult.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\ProductUpgraderInfo.Shared.cs" Link="Common\ProductUpgraderInfo.Shared.cs" />
+    <Compile Include="..\GVFS.Common\Tracing\EventLevel.cs">
+      <Link>Common\Tracing\EventLevel.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Tracing\EventMetadata.cs">
+      <Link>Common\Tracing\EventMetadata.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Tracing\EventOpcode.cs">
+      <Link>Common\Tracing\EventOpcode.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Tracing\ITracer.cs">
+      <Link>Common\Tracing\ITracer.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Common\Tracing\Keywords.cs">
+      <Link>Common\Tracing\Keywords.cs</Link>
+    </Compile>
+    <Compile Include="..\GVFS.Platform.Linux\LinuxPlatform.Shared.cs">
+      <Link>"LinuxPlatform.Shared.cs"</Link>
```

Comment:
For consistency with Mac and Windows, link this in as "Linux\LinuxPlatform.Shared.cs"

Reply:
Thanks, fixed!